api-mock/api-mock-server.ts

```diff
@@ -111,68 +285,127 @@ app.use((req, res, next) => {
 });
 app.use(middleware());
 app.use(errorHandler());
-app.use((err, req, res, next) => {
-    log(err);
-    res.status(500).send("error");
-});
 
 app.get("/mock/ping", async (req, res, next) => {
     res.json({ ok: true });
 });
 
-app.post("/mock/reset", async (req, res, next) => {
-    Object.keys(vars).forEach((key) => (vars[key] = ""));
-    initST(req.body);
+app.post("/mock/init", async (req, res, next) => {
+    initST(req.body.config);
     res.json({ ok: true });
 });
 
-app.get("/mock/getMockedValues", async (req, res, next) => {
+app.get("/mock/mockedvalues", async (req, res, next) => {
```

Comment:
I'd prefer if we didn't call these mocked values/vars, because that's a bit misleading (it could also means the values we've set in mocks). We could call them `overrideparams` for example, but I'm not sure if that's the best option.
If we are going to go in the "override call log" direction we've discussed yesterday, then you do not need to worry about renaming this.

Reply:
agreed, renamed